pkg/osquery/extension.go

```diff
@@ -786,6 +797,19 @@ func (e *Extension) writeResultsWithReenroll(ctx context.Context, results []dist
 }
 
 func getEnrollDetails(client Querier) (service.EnrollmentDetails, error) {
+	var details service.EnrollmentDetails
+
+	if os.Getenv("LAUNCHER_DEBUG_ENROLL_DETAILS_ERROR") == "true" {
```

Comment:
is this just something you're using to simulate errors while querying for enrollment details? Or is there some plan to use it during debugging to reduce the time launcher moves on from trying to get enrollment details?